backend/dataline/services/llm_flow/toolkit.py

```diff
@@ -137,14 +136,13 @@ class ToolNames:
     GENERATE_CHART = "generate_chart"
 
 
-class BaseSQLDatabaseTool(BaseModelV1):
+class BaseSQLDatabaseTool(BaseModel):
     """Base tool for interacting with a SQL database."""
 
-    # TODO: Customize SQLDatabase class to add our improvements
-    db: SQLDatabase = Field(exclude=True)
+    db: Annotated[SQLDatabase, SkipValidation] = Field(exclude=True)
 
-    class Config(BaseTool.Config):
-        pass
+    class Config:
+        arbitrary_types_allowed = True
```

Comment:
Did we need to inherit from `BaseTool.Config`?

Reply:
Good question. I think the real answer is that this entire tool should inherit from BaseTool.

Config can remain like this.